src/Text/Digestive/Aeson.hs

```diff
@@ -74,7 +74,7 @@ Example:
 -}
 jsonErrors :: ToJSON a => View a -> Value
 jsonErrors v =
-  fromMaybe (error "Unable to construct error response")
+  fromMaybe (object [])
```

Comment:
When there are some errors to report, the generated json is a json object whose fields correspond to the problematic fields. So an empty json object is quite a natural value to return when there are no validation errors to report.